modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/configuration/CTConfigurationRegistry.java

```diff
@@ -32,26 +32,26 @@
 public class CTConfigurationRegistry {
 
 	public Optional<CTConfiguration<?, ?>>
-		getCTConfigurationByResourceClass(Class<?> clazz) {
+		getCTConfigurationOptionalByResourceClass(Class<?> clazz) {
```

Comment:
@brianchandotcom Optional word is added here.

Logic is: getting an Optional of a CTConfiguration by the resource class